src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -308,6 +308,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="6.18.0" />
```

Comment:
Did you need to update this?

Reply:
Yes pulling in the ITelemetryClient from this.